.github/workflows/anaconda_pkg_build_linux_alma.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and publish AlmaLinux 8 package builder images
+on:
+  #push:
+  #  branches:
+  #    - main
+  #  tags:
+  #    - 'pkg-build-*'
+  #  paths:
+  #    - 'anaconda-pkg-build/linux/alma/Dockerfile'
+  #    - '.github/workflows/anaconda_pkg_build_linux_alma.yml'
+  #pull_request:
+  #  paths:
+  #    - 'anaconda-pkg-build/linux/alma/Dockerfile'
+  #    - '.github/workflows/anaconda_pkg_build_linux_alma.yml'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+        if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags')
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to Public ECR
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+        if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags')
+        with:
+          registry: public.ecr.aws
+          username: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        env:
+          AWS_REGION: us-east-1
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@4574d27a4764455b42196d70a065bc6853246a25 # v3.4.0
+        with:
+          # https://github.com/docker/setup-qemu-action/issues/188#issuecomment-2604322104
+          image: tonistiigi/binfmt:qemu-v8.1.5
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@f7ce87c1d6bead3e36075b2ce75da1f6cc28aaca # v3.9.0
+        with:
+          version: latest
+          driver-opts: network=host
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@369eb591f429131d6889c46b94e711f089e6ca96 # v5.6.1
+        with:
+          images: |
+            continuumio/anaconda-pkg-build
+            public.ecr.aws/y0o4y9o3/anaconda-pkg-build
+          tags: |
+            type=ref,event=branch,suffix=-almalinux-8
+            type=ref,event=pr,suffix=-almalinux-8
+            type=match,pattern=pkg-build-(.*),group=1,suffix=-almalinux-8
```

Comment:
My memory here is fuzzy, @JeanChristopheMorinPerso can you confirm this won't do a `latest` tag based on: https://github.com/docker/metadata-action?tab=readme-ov-file#latest-tag

Reply:
Though, it does say that it would be enalbled for [type=match,pattern=...](https://github.com/docker/metadata-action?tab=readme-ov-file#typematch)...

Reply:
We could force `latest= false ` in `flavors`

Reply:
Yeah, here is what we did in the past for another project: https://github.com/anaconda/docker-images/blob/752b2d84e020c93eaa5b27631fb20e1f5cf2f636/.github/workflows/anaconda_pkg_build_lts_linux.yml#L61-L62.

@danpetry Can you make that change please?